### PR TITLE
Feat/bp page styles

### LINKED ISSUE #809: BP page styles
Please take a look at this frame: https://zpl.io/VOEyReY
Review main card paddings, text specs, tabulation and line spacing. Thanks.

### PR DESCRIPTION
**Pull request by hanakoori01**
### What does this PR do?

- Resolves #809 

### Steps to test

1. Run `make run`
2. Go to `localhost:3000`
3. Go to Block Producers > Block Producer Profile 

#### Preview Images of changes
- Desktop
BP_1
BP_2

- Mobile
BP_3

### CODE REVIEW

**Review by AngeloCG97 — COMMENTED** [review]
Please change

``` 
<TableBoxColor color={polarChartData[0].color}></TableBoxColor>
```
by 
``` 
<TableBoxColor color={polarChartData[0].color} />
```

File: webapp/src/components/TableBoxColor/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import styles from './styles'
+import { LightenDarkenColor } from 'lighten-darken-color'
+
+const useStyles = makeStyles(styles)
+
+const TableBoxColor = ({ color }) => {
+  const classes = useStyles()
+  return (
+    <Box
+      style={{ borderColor: LightenDarkenColor('#a43bca99', 80) }}
```
Comment (AngeloCG97) [inline webapp/src/components/TableBoxColor/index.js:15]: Please remove inline style and change by Box props

``` 
borderColor={ LightenDarkenColor('#a43bca99', 80) }
```

**Review by tetogomez — APPROVED** [review]
LGTM

File: webapp/src/routes/BlockProducers/BlockProducerProfile.js [hunk 2]
```diff
@@ -305,45 +305,54 @@ const BlockProducerProfile = () => {
...
-                  amount:
-                    _get(state.blockProducer, 'total_ratings_cntr', null) || 0,
-                  average: getAverageValue(
-                    _get(state.blockProducer, 'total_average', 0)
-                  )
-                }
-              ]}
-              heads={[t('raters'), t('amount'), t('average')]}
-            />
+            {console.log(polarChartData)}
```
Comment (AngeloCG97) [inline webapp/src/routes/BlockProducers/BlockProducerProfile.js:308]: Please remove this console.log

**Review by AngeloCG97 — CHANGES_REQUESTED** [review]

**Review by AngeloCG97 — CHANGES_REQUESTED** [review]

**Review by AngeloCG97 — APPROVED** [review]
LGTM

**Pull request merged**